rocketpool-cli/node/commands.go

```diff
@@ -340,10 +340,9 @@ func RegisterCommands(app *cli.App, name string, aliases []string) {
 
 					// Validate flags
 					if c.String("amount") != "" &&
-						c.String("amount") != "min8" &&
-						c.String("amount") != "max8" &&
-						c.String("amount") != "min16" &&
-						c.String("amount") != "max16" &&
+						c.String("amount") != "5%" &&
+						c.String("amount") != "10%" &&
+						c.String("amount") != "15%" &&
```

Comment:
Can we drop the magic values and just use `strings.EndsWith(c.String("amount"), "%")` to determine if a percentage was passed? Then parse it and the actual amount.

Reply:
Good call. The latest commit allows for custom percentages passed through as CLI flags and additionally as an option within the command.